src/main/kotlin/com/funivan/idea/phpClean/inspections/redundantDocCommentTag/RedundantDocCommentTagInspection.kt

```diff
@@ -37,6 +39,19 @@ class RedundantDocCommentTagInspection : PhpCleanInspection() {
                 }
             }
 
+            override fun visitPhpField(field: Field) {
+                val comment = field.docComment
+                val items = mutableListOf<ParameterType>()
+                if (comment != null) {
+                    for (paramTag in comment.paramTags) {
```

Comment:
Why you are collecting items, and then iterate over this collection. Maybe you can parameter types immediately?
```kotlin
for (paramTag in comment.paramTags) {
   checkComment(paramTag, field. declaredType)
}
```